### PR TITLE
python3Packages.aioapns: init 2.0.2

### PR DESCRIPTION
**Pull request by superherointj**
* python3Packages.aioapns: init 2.0.2

This is a dependency of `sygnal` (#137092).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 139322` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.aioapns</li>
    <li>python39Packages.aioapns</li>
  </ul>
</details>

File: pkgs/development/python-modules/aioapns/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    inherit pname version;
+    sha256 = "ea58ce685aa6d0ffbdc3be4a6999c7268b9c765f806d3e4da7677c098fb72cbc";
+  };
+
+  propagatedBuildInputs = [
+    h2
+    pyopenssl
+    pyjwt
+  ];
+
```
Comment (fabaff) [inline pkgs/development/python-modules/aioapns/default.nix:23]: As there are no tests.

```suggestion
  # Project has no tests
  doCheck = false;
```

**Review by fabaff — COMMENTED** [review]
Please update the commit message.

```bash
python3Packages.aioapns: init 2.0.2
```

File: pkgs/development/python-modules/aioapns/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/python-modules/aioapns/default.nix:23]: Done

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 139322` at 868a8a52 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.aioapns</li>
    <li>python39Packages.aioapns</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 139322` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.aioapns</li>
    <li>python39Packages.aioapns</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package builds (see https://github.com/NixOS/nixpkgs/pull/139322#issuecomment-926924300)
- [ ] all depending packages build
- [x] tested on x86_64

Looks good to me.

**Pull request merged**